### PR TITLE
II-3810 csv to json log parser

### PR DESCRIPTION
**Pull request by crespo5609**
[ticket](https://insightfinders.atlassian.net/browse/II-3810)

### CODE REVIEW

File: csv_to_json_log_parser/config.ini [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[Config]
+tag = cmdb_application
```
Comment (yqian10) [inline csv_to_json_log_parser/config.ini]: please remove the information here, then change the file to config.ini.template

File: csv_to_json_log_parser/parseData.py [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+def get_parameters():
+    usage = "Usage: %prog [options]"
+    parser = OptionParser(usage=usage)
+    parser.add_option("-d", "--directory",
+                      action="store", dest="homepath", help="Directory to run from")
+    parser.add_option("-w", "--server_url",
+                      action="store", dest="server_url", help="Server Url")
+    parser.add_option("-l", "--chunk_lines",
+                      action="store", dest="chunk_lines", help="Max number of lines in chunk")
+    parser.add_option("-m", "--max_in_tag",
```
Comment (yqian10) [inline csv_to_json_log_parser/parseData.py]: please change tag to some more meaningful name e.x. max_number_in_tag

File: csv_to_json_log_parser/parseData.py [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    #epoch_time = (utc_time - datetime(1970, 1, 1)).total_seconds()
+    return epoch
+
+def get_agent_config_vars():
+    config_vars = {}
+    try:
+        if os.path.exists(os.path.join(parameters['homepath'], "config.ini")):
+            parser = SafeConfigParser()
+            parser.read(os.path.join(parameters['homepath'], "config.ini"))
+            tag = parser.get(directory, 'tag')
```
Comment (yqian10) [inline csv_to_json_log_parser/parseData.py]: please remove the unsed parameter

File: csv_to_json_log_parser/parseData.py [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+            file_name = parser.get(directory, 'file_name')
+            license_key = parser.get(directory, 'insightFinder_license_key')
+            project_name = parser.get(directory, 'insightFinder_project_name')
+            user_name = parser.get(directory, 'insightFinder_user_name')
+            sampling_interval = parser.get(directory, 'sampling_interval')
+            timestamp = parser.get(directory, 'timestamp_key')
+            timestamp_pattern = parser.get(directory, 'timestamp_pattern')
+            instance_name = parser.get(directory, 'instance_name_key')
+            keys_to_filter = json.loads(parser.get(directory, 'keys_to_filter'))
+            if len(tag) == 0:
```
Comment (yqian10) [inline csv_to_json_log_parser/parseData.py]: remove here

File: csv_to_json_log_parser/parseData.py [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+            if len(timestamp) == 0:
+                logger.error("Agent not correctly configured(timestamp). Check config file.")
+                sys.exit(1)
+            if len(timestamp_pattern) == 0:
+                logger.error("Agent not correctly configured(timestamp pattern). Check config file.")
+                sys.exit(1)
+            if len(instance_name) == 0:
+                logger.error("Agent not correctly configured(instance name). Check config file.")
+                sys.exit(1)
+            config_vars[tag_key] = tag
```
Comment (yqian10) [inline csv_to_json_log_parser/parseData.py]: config_vars[tag_key] is not used.

File: csv_to_json_log_parser/config.ini [hunk 6]
```diff
@@ -0,0 +1,11 @@
+[Config]
```
Comment (gnegri) [inline csv_to_json_log_parser/config.ini]: remove your project's info and rename to config.ini.template

**Pull request merged**